addons/hr_timesheet/wizard/project_task_create_timesheet.py

```diff
@@ -20,13 +21,19 @@ class ProjectTaskCreateTimesheet(models.TransientModel):
     )
 
     def save_timesheet(self):
+        minimum_duration = int(self.env['ir.config_parameter'].sudo().get_param('hr_timesheet.timesheet_min_duration', 0))
+        rounding = int(self.env['ir.config_parameter'].sudo().get_param('hr_timesheet.timesheet_rounding', 0))
+        minutes_spent = max(minimum_duration, self.time_spent * 60)
+        if rounding and ceil(minutes_spent % rounding) != 0:
+            minutes_spent = ceil(minutes_spent / rounding) * rounding
+
         values = {
             'task_id': self.task_id.id,
             'project_id': self.task_id.project_id.id,
             'date': fields.Date.context_today(self),
             'name': self.description,
             'user_id': self.env.uid,
-            'unit_amount': self.time_spent,
+            'unit_amount': minutes_spent / 60,
         }
         self.task_id.user_timer_id.unlink()
```

Comment:
🙈 Can you comment this line?
And add a comment above saying that the wizard has to be moved to timesheet_grid in master?

Reply:
And add an inheritance in timesheet_grid that call super and call self....
Thanks!

Reply:
This wizard should have been moved to enterprise at the timer has been moved to enterprise.